src/d3-ext/profile.js

```diff
@@ -58,13 +62,13 @@ ngeo.profile = function() {
    * @type {function(Object)}
    * Hover callback function.
    */
-  var hoverCallback = goog.nullFunction;
+  var hoverCallback = options.hoverCallback || goog.nullFunction;
```

Comment:
I think we want to be consistent and use `goog.isDef` everywhere. Much easier to explain to new developers.